react-native-libraries.json

```
@@ -9818,10 +9818,21 @@
     "web": true,
     "expo": true
   },
-    {
+  {
     "githubUrl": "https://github.com/Malaa-tech/react-native-simple-keypad",
     "examples": ["https://github.com/Malaa-tech/react-native-simple-keypad/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/chsdwn/react-native-responsive-scalability",
+    "examples": [
+      "https://github.com/chsdwn/react-native-responsive-scalability/tree/main/example",
+      "https://github.com/chsdwn/FreeToGameApp"
```

Hello @chsdwn, thank you for adding new entry to the directory, LGTM! 👍 

Just one nit, let's remove "FreeToGameApp" example since this is mor a demo app, than an library usage example which can be easy understood by users. We might explore adding new sub page and data point for React Native based open source apps to the directory in the future (which will be a great fit for that app).

```suggestion
      "https://github.com/chsdwn/react-native-responsive-scalability/tree/main/example"
```